about.css

```diff
@@ -0,0 +1,953 @@
+    background-color: #ec5242;
+    opacity: 0.6;
+    margin: 15px 0 30px 0;
+  }
+
+  #pastevents p {
+    text-align: center;
+    font-size: 0.8rem;
+    width: 80%;
+  }
+
+  main #logo-section p {
+    font-size: 0.8rem;
+    width: 80%;
+    text-align: center;
+  }
+
+  #logo-section .logo-container {
+    border: 2px solid #eeeded;
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    gap: 3px;
+    margin-top: 30px;
+    padding: 20px 40px;
+    width: 80%;
+  }
+
+  .logo-container span {
+    color: #ec5242;
+  }
+
+  .logo-container div {
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+  }
+
+  .logo-container div span {
+    color: black;
+    font-family: cocogoose, sans-serif;
+  }
+
+  #pastevents {
+    padding: 40px 30px;
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+  }
+
+  #pastevents h2 {
+    text-align: center;
+    font-size: 1.2rem;
+  }
+
+  #pastevents .containers {
+    width: 100%;
+    height: 200px;
+    margin-top: 20px;
+  }
+
+  #pastevents .containers .red-div {
+    background-color: rgba(187, 32, 32, 0.5);
+    height: 100%;
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    padding: 0 6px;
+  }
+
+  .red-div h4 {
+    color: #fff;
+    font-size: 1.5rem;
+    margin-bottom: 15px;
+    font-family: cocogoose, sans-serif;
+  }
+
+  #pastevents .containers .red-div span {
+    color: #fff;
+  }
+
+  #pastevents #container-1 {
+    background-image: url(./images/past1.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+  }
+
+  #pastevents #container-2 {
+    background-image: url(./images/past2.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+  }
+
+  footer {
+    display: flex;
+    justify-content: space-between;
+    width: 90%;
+    margin-left: 0;
+  }
+}
+
+@media (min-width: 768px) {
+  nav {
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    width: 90%;
```

Comment:
- [ ] Same here on the about page kindly make the header take the whole page width.

Reply:
Thank you. Done.